gulpfile.js

```diff
@@ -25,11 +25,15 @@ gulp.task( 'docs', () => {
 			sourceFiles: [
 				process.cwd() + '/packages/ckeditor5-*/src/**/*.@(js|jsdoc)',
 				'!' + process.cwd() + '/packages/ckeditor5-*/src/lib/**/*.js'
-			],
-			destinationPath: path.join( process.cwd(), 'build', 'docs' )
+			]
 		} );
 } );
 
+gulp.task( 'umberto', () => {
+	return require( 'umberto' )
+		.buildSingleProject( true );
```

Comment:
Maybe it's not a part of this PR, but we used to passing objects instead of flags as the objects are more explicit in such situations and easier to understand.

Consider this:
```js
.buildSingleProject( { clean: true } );
```